### PR TITLE
Uidatimp 572

### PR DESCRIPTION
**Pull request by IvanKryzhanovskyi**
## Purpose
Reuse <JobLogs> component from data-transfer-components rep

## Approach
- Existing JobLogs component has been removed
- RecentJobLogs and JobsLogsContainer has been created

## Refs
https://issues.folio.org/browse/UIDATIMP-572

### CODE REVIEW

File: src/components/JobLogsContainer/JobLogsContainer.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  useJobLogsListFormatter,
+} from '@folio/stripes-data-transfer-components';
+import {
+  Button,
+  Callout,
+} from '@folio/stripes/components';
+import { listTemplate } from '../ListTemplate';
+import sharedCss from '../../shared.css';
+
+import { DEFAULT_JOB_LOG_COLUMNS } from '../../utils';
```
Comment (mariia-aloshyna) [inline src/components/JobLogsContainer/JobLogsContainer.js:16]: Put an empty line between imports from folio, own components imports, and styles imports, please.
See this wiki page for more details - https://wiki.folio.org/pages/resumedraft.action?draftId=14458784&draftShareId=483fa2fb-615e-4002-a916-c976bd5fb8c6&

```suggestion
} from '@folio/stripes/components';

import { listTemplate } from '../ListTemplate';
import { DEFAULT_JOB_LOG_COLUMNS } from '../../utils';

import sharedCss from '../../shared.css';
```

File: src/components/JobLogsContainer/JobLogsContainer.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        marginBottom0
+        to={`/data-import/log/${record.id}`}
+        buttonClass={sharedCss.cellLink}
+        target="_blank"
+        onClick={e => e.stopPropagation()}
+      >
+        {record.fileName}
+      </Button>
+    );
+  };
```
Comment (mariia-aloshyna) [inline src/components/JobLogsContainer/JobLogsContainer.js:58]:
```suggestion
  const fileNameCellFormatter = record => (
      <Button
        buttonStyle="link"
        marginBottom0
        to={`/data-import/log/${record.id}`}
        buttonClass={sharedCss.cellLink}
        target="_blank"
        onClick={e => e.stopPropagation()}
      >
        {record.fileName}
      </Button>
  );
```

File: src/components/JobLogsContainer/JobLogsContainer.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+  const listProps = {
+    ...useJobLogsProperties(customProperties),
+    resultsFormatter: useJobLogsListFormatter(
+      {
+        ...listTemplate({ intl }),
```
Comment (mariia-aloshyna) [inline src/components/JobLogsContainer/JobLogsContainer.js:64]: I don't quite understand why you are passing `intl` to the `listTemplate`?

File: src/components/JobLogsContainer/JobLogsContainer.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        fileName: record => fileNameCellFormatter(record),
```
Comment (mariia-aloshyna) [inline src/components/JobLogsContainer/JobLogsContainer.js:65]:
```suggestion
        fileName: fileNameCellFormatter,
```

File: src/components/RecentJobLogs/RecentJobLogs.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
+import React, { useContext } from 'react';
+
+import {
+  JobLogs,
+  DEFAULT_JOB_LOGS_SORT_COLUMNS,
+  sortStrings,
+} from '@folio/stripes-data-transfer-components';
+
```
Comment (mariia-aloshyna) [inline src/components/RecentJobLogs/RecentJobLogs.js:8]: Remove this empty line, please

File: src/components/RecentJobLogs/RecentJobLogs.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+import { stripesConnect } from '@folio/stripes/core';
+
+import { DataFetcherContext } from '../DataFetcher';
+
```
Comment (mariia-aloshyna) [inline src/components/RecentJobLogs/RecentJobLogs.js:12]: Remove this empty line, please

**Comment by mariia-aloshyna** [comment]
Update the `CHANGELOG`, please

File: src/components/JobLogsContainer/JobLogsContainer.js (hunk 1 above)
Comment (IvanKryzhanovskyi) [reply src/components/JobLogsContainer/JobLogsContainer.js]: done

File: src/components/JobLogsContainer/JobLogsContainer.js (hunk 2 above)
Comment (IvanKryzhanovskyi) [reply src/components/JobLogsContainer/JobLogsContainer.js]: done

File: src/components/RecentJobLogs/RecentJobLogs.js (hunk 5 above)
Comment (IvanKryzhanovskyi) [reply src/components/RecentJobLogs/RecentJobLogs.js]: Done

File: src/components/JobLogsContainer/JobLogsContainer.js (hunk 4 above)
Comment (IvanKryzhanovskyi) [reply src/components/JobLogsContainer/JobLogsContainer.js]: Done

File: src/components/RecentJobLogs/RecentJobLogs.js (hunk 6 above)
Comment (IvanKryzhanovskyi) [reply src/components/RecentJobLogs/RecentJobLogs.js]: Done

File: src/components/JobLogsContainer/JobLogsContainer.js (hunk 3 above)
Comment (IvanKryzhanovskyi) [reply src/components/JobLogsContainer/JobLogsContainer.js]: Done

**Review by IhorKoba — APPROVED** [review]

**Review by mariia-aloshyna — APPROVED** [review]

**Pull request merged**